src/lottie.js

```diff
@@ -10,12 +10,54 @@ document.querySelectorAll( '[data-lottie]' ).forEach( ( lottie ) => {
 		return;
 	}
 
-	// Check if this is a cover block
-	const isCoverBlock = lottie.classList.contains( 'wp-block-cover' );
```

Comment:
This constant definition shouldn't be getting removed, maybe there was a merge conflict here?

Reply:
Restored